.github/workflows/test.yml

```diff
@@ -72,6 +72,8 @@ jobs:
       - name: Read default Xcode version
         run: |
           echo "DEFAULT_XCODE_VERSION=$(cat .xcode-version | tr -d '\n')" >> "$GITHUB_ENV"
+          echo "DEFAULT_SIMULATOR_NAME=$(cat .test-simulator-device-name | tr -d '\n')" >> "$GITHUB_ENV"
+          echo "DEFAULT_SIMULATOR_VERSION=$(cat .test-simulator-ios-version | tr -d '\n')" >> "$GITHUB_ENV"
```

Comment:
🤔 Maybe the name of the step should be updated given that it does more than just the Xcode version. What do you think?

Reply:
🤦🏼‍♀️ It is in the Xcode 16 branch, I just didn't bring it over here